ui/packages/security-idp-server/src/main/webapp/js/view/Login.view.js

```diff
@@ -99,7 +114,7 @@ define([
     clearFields: function() {
       this.$('#username').val('')
       this.$('#password').val('')
-      this.$('#loginError').hide()
+      this.$('#loginerrorup').hide()
```

Comment:
❓ So if the `clear` button is clicked, we hide the `invalid credentials` error message. Should we also hide the error message when one of the input fields is changed?